src/site/stages/build/drupal/api.js

```diff
@@ -108,6 +115,28 @@ function getDrupalClient(buildOptions) {
       });
     },
 
+    getRemainingContent(onlyPublishedContent = true) {
+      return this.query({
+        query: getQuery(queries.GET_REMAINING_CONTENT),
+        variables: {
+          onlyPublishedContent,
+        },
+      });
+    },
+
+    getExportedPages() {
+      const exportDir = buildOptions['cms-export-dir'] || contentDir;
```

Comment:
```suggestion
      const exportDir = buildOptions['cms-export-dir'] || contentDir;
      const assembleEntityTree = entityTreeFactory(exportDir);
```
For reasons stated above.